cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -197,7 +198,8 @@ def set_agent_creds(appliance, request, provider):
 def local_setup_provider(request, setup_provider_modscope, provider, appliance):
     # TODO: allow for vddk parameterization
     if provider.one_of(VMwareProvider):
-        appliance.install_vddk()
+        vddk_url = conf.cfme_data.get("basic_info").get("vddk_url").get('v6_0')
```

Comment:
When chaining these `.get()` calls, you need to have a default of an empty dictionary so that the subsequent calls don't hit an `AttributeError`.

Additionally, I think this should handle the case where it doesn't find a URL by skipping the test.

```
vddk_url = conf.cfme_data.get('basic_info', {}).get('vddk_url', {}).get('v6_0', None)
if vddk_url is None:
    pytest.skip('Could not locate vddk url in cfme_data')
else:
    appliance.install_vddk(vddk_url)
```

Reply:
Updated.